apache-ansible-demo/docs/adopt.md

```diff
@@ -0,0 +1,135 @@
+```
+git clone https://github.com/IBM/tnco-package-examplesCopy code
+```
+
+2. Create an OpenStack stack that can be adopted by an assembly:  
+```
+cd tnco-package-examples/apache-ansible-demo/Contains/hw-apache-vnfc/Lifecycle/scripts/Copy code
+```
+
+Edit the script file create.sh and customize the uncommented variables in the script:
+```
+OS_URL=<openstack-server-url>
+OS_AUTH_PROJECT_NAME=<openstack-project>
+OS_AUTH_USERNAME=<openstack-username>
+OS_AUTH_PASSWORD=<password>Copy code
+```
+
+Where:
+   * OS_URL is the URL to the OpenStack server.
+   * OS_AUTH_PROJECTNAME is the name of the OpenStack project that is associated with the OpenStack user.
+   * OS_AUTH_USERNAME and OS_AUTH_PASSWORD are the username and password of the OpenStack user.
+
+Depending on the OpenStack environment, you might need to set other variables in the script that are commented out. For example, if the OpenStack environment is set up to support Transport Layer Security (TLS), you might need to set the variable OS_CACERT_FILE to point to a file that stores the OpenStack CA certificates.
+
+Run the create.sh script:
+```
+./create.sh
+```
+  
+The script runs the ansible playbooks contained in the resource package to create an OpenStack stack with infrastructure resources to host an Apache server. The Apache server is then configured and started.
+  
+3. Verify that the OpenStack stack is created.  
+   1. Log in to the OpenStack dashboard with your OpenStack username and password.  
+   2. Click the project menu and select the OpenStack project that you specified in the create.sh script.  
+   3. From the Orchestration menu, select Stacks. You see a new stack with a name that starts with apache_ansible.  
+  
+4. Save the stack ID in a variable to use as an input parameter for the Adopt REST API.  
+   1. In the OpenStack dashboard, click the created stack, and then click Overview.  
+   2. Copy the stack ID, and use it to set the STACK_ID environment variable:  
+   ```
+   export STACK_ID=<stack_id>
+   ```
+
+5. Verify that the Apache server is up and running:  
+   1. From the stack Overview, collect the value of the public_ip parameter. The public_ip parameter is the IP address that is associated with the Apache server on the public network.  
+   2. Paste the IP address into a web browser that has connectivity to the OpenStack public network.  
+   3. Confirm the Apache server is running. You see the following message in the web browser:  
+   ```
+   Hello, world
+   Success!
+   ```
+
+6. Set up the environment to invoke the IBM Telco Network Cloud - Orchestration REST API.  
+   1. Set the TNCO_URL environment variable to point to the API URL:  
+   ```
+   export TNCO_URL=<ishtar_route>
+   ```  
+   2. Set the TNCO_TOKEN environment variable to point to the access token
+   ```
+   export TNCO_TOKEN=<access_token>
+   ```
+   
+   For more information about how to retrieve the ishtar_route and access_token parameters, see Invoking REST API.
```

Comment:
Should we add a link here to the KC section for Invoking the REST API
https://www-03preprod.ibm.com/support/knowledgecenter/SSDSDC_1.3/Installing/c_alm_invoking_rest_api.html